agenta-cli/agenta/sdk/types.py

```diff
@@ -98,6 +98,32 @@ def __modify_schema__(cls, field_schema: dict[str, Any]):
         )
 
 
+class MessagesParam(list):
```

Comment:
Is there a way to have this as a pydantic class I wonder? This would allow us to make sure the schema is correct

Reply:
```
# Define a model for the individual message
class Message(BaseModel):
    role: str
    content: str

class MessageList(BaseModel):
    messages: List[Message]

    @classmethod
    def __modify_schema__(cls, field_schema: dict[str, Any]):
        field_schema.update({"x-parameter": "messages"})
```

Reply:
Thank you sharing this!